ssp

```diff
@@ -8826,6 +8828,22 @@ sub check_for_cpro {
 
 ## END malware checks
 
+sub detect_slowloris {
+    my $sockcount = 75;
```

Comment:
Is just checking for >75 established connections the most robust way to check for Slowloris?  Busy servers can hit this mark easily while not under attack.